### DISCUSSION TITLE
Any plans to break `nio` into a luarocks package?

### DISCUSSION THREAD

**Discussion by mrcjkb** [nvim-neotest/neotest#332] [General]
Hey @rcarriga 👋 

We've been using `nio` in [rocks.nvim](https://github.com/nvim-neorocks/rocks.nvim) quite happily for a while now.

When you switched from `plenary.async` to `nio`, I recall you mentioning that you had plans to break it out into a separate library.
Are you still interested in doing so?

Cheers 😃

**Comment by rcarriga** [comment]
Ah I hadn't realised anyone was using it, thats great! :sweat_smile:

I've split it out to a new repo here https://github.com/nvim-neotest/nvim-nio and setup the luarocks upload. Looks to be working as expected :raised_hands: 

I'll take it out of neotest (and nvim-dap) soon but just want to make sure it's ready to go.

**Reply by mrcjkb** [reply]
That's awesome! Thanks :pray:
